package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "node index.js",
     "lint": "eslint test",
-    "test": "mocha test/**/*-test.js",
+    "test": "eslint test && mocha test/**/*-test.js",
```

Comment:
we can do not repeat ourselves and write
`"test": "npm run lint && mocha test/**/*-test.js",`